File: tests/test_scala/com/example/services/OrderService.scala
package com.example.services

import com.example.models.{Order, User}
import java.time.LocalDateTime

class OrderService {
    def process(order: Order, user: User): Boolean = {
        println(s"Processing order ${order.id} for user ${user.name}")
        println(s"Total: ${order.formatTotal}")
        true
    }

    def cancel(orderId: Long): Boolean = {
        println(s"Canceling order $orderId at ${LocalDateTime.now()}")
        true
    }

    def getOrdersByUser(user: User): List[Order] = {
        List.empty[Order]
    }
}

File: tests/test_kotlin/com/example/services/OrderService.kt
package com.example.services

import com.example.models.Order
import com.example.models.User
import java.time.LocalDateTime

class OrderService {
    fun process(order: Order, user: User): Boolean {
        println("Processing order ${order.id} for user ${user.name}")
        println("Total: ${order.formatTotal()}")
        return true
    }

    fun cancel(orderId: Long): Boolean {
        println("Canceling order $orderId at ${LocalDateTime.now()}")
        return true
    }

    fun getOrdersByUser(user: User): List<Order> {
        return emptyList()
    }
}
